web/atlas.json

```diff
@@ -761,7 +761,7 @@
+{"id": "twm4ix", "submitted_by": "Various_Profit1097", "name": "Reimu Hakurei & Hatsune Miku heart", "description": "An heart between a small Reimu and a small Miku symbolizing the friendship between the two subreddits throughout r/place 2022", "website": "", "subreddit": "/r/touhou, /r/hatsune", "center": [810.5, 697.5], "path": [[807.5,704.5], [804.5,701.5], [806.5,699.5], [807.5,700.5], [808.5,699.5], [810.5,701.5]] },

```

Comment:
The center is no longer on the entry. Redraw the path and remember to replace the center as well.